docs/pages/grafana.mdx

```diff
@@ -0,0 +1,217 @@
+
+Append the following lines to your `pelican.yaml` file:
+
+```yaml filename="pelican,yaml"
+# Other configurations ...
+
+Monitoring:
+    PromQLAuthorization: false
+```
+
+> Note that you need to restart the Pelican server to apply the changes.
+
+#### Set through environment variable
+
+The environment variable for the configuration is `PELICAN_MONITORING_PROMQLAUTHORIZATION`.
+
+You can either `export` this environment variable or append it before your command to run Pelican.
+
+```bash
+export PELICAN_MONITORING_PROMQLAUTHORIZATION=false
+```
+
+or
+
+```bash
+PELICAN_MONITORING_PROMQLAUTHORIZATION=false pelican origin serve
+```
+
+Once you configured the authentication either in Grafana or in Pelican, click **Save & Test** to see if the connection can be established.
+
+# Build Monitroing Dashbord
+
+With Grafana connected to Pelican Prometheus instance, you are able to build visualization with the metrics Pelican collects. This section describes how to get started with a template dashboard configuration for a _Pelican origin server_. For a detailed guide on how to create a Grafana dashboard, refer to [Grafana documentation](https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/create-dashboard/).
+
+## Before start
+
+1. Make sure your Pelican origin server is running
+
+2. Download the template dashboard file from [here](https://github.com/PelicanPlatform/pelican/blob/main/docs/public/origin-dashboard-template.json)
+
+
+## Modify dashboard template
+
+There are a couple of placeholders need to be changed before you can import the dashboard template to Gafana. To do a batch replace, it is recommended that you use a text editor to find and replace placeholders to corresponding values.
+
+* Replace `<YOUR-DATA-SOURCE-UID>` to the id of your Prometheus data source. You can find the ID on the data source configuration page.
+
+    1. Navigate to the configuration page
+    2. Copy the ID from your browser's URL box, which should have a value similar to
+
+        ```console
+        http://localhost:3000/connections/datasources/edit/fc2115e0-ede3-4fdb-9056-ecefa1b3160a
+        ```
+
+        Where `fc2115e0-ede3-4fdb-9056-ecefa1b3160a` is the ID you want to copy. Note that the ID in the example above will be different from your data source ID
+
+    3. Batch replace `<YOUR-DATA-SOURCE-UID>` in the template dashboard file to the copied value
+
+* Batch replace `<YOUR-ORIGIN-SERVER-WEB-URL>` to the web URL of your Pelican origin server, without the `https://` portion. Example: `localhost:8444`
+
+* Save the changes to your template dashboard file.
+
+## Import template file to Grafana
+
+1. Go to home page of Grafana
+2. Click **Dashboards** in the left-side menu. If there is no menu on the left-side, click the icon of stacked bars to toggle the menu, which is located on the top-left side of the page next to **Home**
```

Comment:
```suggestion
2. Click **Dashboards** on the left-side menu. If there is no menu on the left-side, click the icon of stacked bars to toggle the menu, which is located on the top-left side of the page next to **Home**
```

Reply:
Grafana documentation uses `in` instead of `on` so I'm keeping up with the same style here